js/content/community.js

```diff
@@ -3029,6 +3029,61 @@ let FriendsPageClass = (function(){
     return FriendsPageClass;
 })();
 
+let GroupsPageClass = (function(){
+
+    function GroupsPageClass() {
+        this.addSort();
+    }
+
+    GroupsPageClass.prototype.addSort = function() {
+        let groups = Array.from(document.querySelectorAll(".group_block"));
+        if (groups.length === 0) { return; }
+
+        groups.forEach((group, i) => group.dataset.esSortdefault = i);
+
+        function sortGroups(sortBy, reversed) {
+            for (let group of groups) {
+                let name = group.querySelector(".groupTitle > a").textContent;
+                let membercount = Number(group.querySelector(".memberRow > a").textContent.match(/\d+/g).join(""));
+                group.dataset.esSortnames = name;
+                group.dataset.esSortmembers = membercount;
+            }
+
+            let searchResults = document.querySelector("#search_results_empty");
+            let property = `esSort${sortBy.toLowerCase()}`;
+            groups.sort((a, b) => {
+                let propA = a.dataset[property];
+                let propB = b.dataset[property];
+                if (isNaN(propA)) {               
+                    if (propA.toLowerCase() < propB.toLowerCase()) { return -1; }
+                    if (propA.toLowerCase() > propB.toLowerCase()) { return 1; }
+                    return 0;
+                }
+                return Number(propA) - Number(propB);
+            });
+
+            for (let group of groups) {
+                if (reversed) {
+                    searchResults.insertAdjacentElement("afterend", group);
+                } else {
+                    searchResults.parentElement.appendChild(group);
+                }
+            }
+        }
+
+        let sortBy = SyncedStorage.get("sortgroupsby") || "default";
+        document.querySelector("#search_text_box").insertAdjacentElement("beforebegin", Sortbox.get(
+            "groups",
+            [["default", Localization.str.datecreated], ["members", Localization.str.members], ["names", Localization.str.name]],
```

Comment:
The default sorting is not sorted by the date created, it's sorted after the date that the user joined.
Either way, I'd stick with "Default" here, since the other sortboxes also do so

Reply:
Wrong, it's sorted by group id, which is also the creation date. I think using "default" robs users from this useful information.

Reply:
![image](https://user-images.githubusercontent.com/14999077/71479369-6b10fb80-27f4-11ea-9080-7b778f3c0e60.png)
I have ITAD as a featured group, so this one stays on top.
You really should name it "Default"

Reply:
Alright, fair point. Still shame we essentially loose the date created filter, but changing to 'default' is the easiest solution. Fixed.

Reply:
Alright, fair point. Still shame we essentially loose the date created filter, but changing to 'default' is the easiest solution. Fixed.